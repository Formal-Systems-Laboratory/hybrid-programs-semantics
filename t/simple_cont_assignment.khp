vars x, v, a ; x ' = v & x <= 10
