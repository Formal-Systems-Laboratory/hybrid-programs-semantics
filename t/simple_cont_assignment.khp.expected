  #Not ( #Not ( {
    ?INITIAL +Int ?INITIAL0 *Int ?T <=Int 10
  #Equals
    true
  } ) )
#And
  <generatedTop>
    <k>
      .
    </k>
    <state>
      a |-> ?INITIAL1:Int
      t |-> ?T:Int
      v |-> ?INITIAL0:Int
      x |-> ?INITIAL +Int ?INITIAL0 *Int ?T
    </state>
  </generatedTop>
#And
  {
    ?INITIAL +Int ?INITIAL0 *Int ?T <=Int 10
  #Equals
    true
  }
